criu/kerndat.c

```diff
@@ -986,6 +987,40 @@ static int kerndat_tun_netns(void)
 	return check_tun_netns_cr(&kdat.tun_ns);
 }
 
+static bool kerndat_has_clone3_set_tid(void)
+{
+	pid_t pid;
+	struct _clone_args args = {};
+
+#ifndef CONFIG_X86_64
+	kdat.has_clone3_set_tid = false;
+	return 0;
+#endif
+
+	args.set_tid = -1;
+	/*
+	 * On a system without clone3() this will return ENOSYS.
+	 * On a system with clone3() but without set_tid this
+	 * will return E2BIG.
+	 * On a system with clone3() and set_tid it will return
+	 * EINVAL.
+	 */
+	pid = syscall(SYS_clone3, &args, sizeof(args));
```

Comment:
You define clone3 in plugins/std/syscalls, in this case you need to use __NR_clone3 instead of SYS_clone3.

And you will need to include  "compel/plugins/std/syscall-codes.h"